slsReceiverSoftware/src/GeneralData.h

```diff
@@ -490,6 +497,7 @@ class Gotthard2Data : public GeneralData {
         vetoDataSize = 160;
         vetoPacketSize = headerSizeinPacket + vetoDataSize;
         vetoImageSize = vetoDataSize * packetsPerFrame;
+        vetoHsize = 24;
```

Comment:
Same here don't we have the header somewhere so that we could use sizeof?

Reply:
No, its in the sever definitions.  Not accsessible here.